src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,118 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Used to load/store TaskList from/to a duke.txt data file.
+ */
+public class Storage {
+
+    private final String dataPath = System.getProperty("user.dir") + "/data";
```

Comment:
Final fields should be all capsed?

Reply:
final and constant are not the same thing though, typically constants refer to class members, final just means it cannot be modified after instantiation. I think it's fine that they are not all capitalised since they are instance fields, however perhaps `fileName` could be a class member, and then it should be in all caps.